/*
  Vehicle Data

  This API section holds messages related to OTA Vehicle Data between device and backend.

  The version of the OpenAPI document: 2

  Generated by OpenAPI Generator: https://openapi-generator.tech
*/

syntax = "proto3";


message DeviceToBackendMessageMeta {

  // If this message is sent in the context of an activity (a functional process with a defined lifecycle tracked by the backend), the ID of that activity shall be put into the activity ID field. If this field is null or missing, the message is not part of an activity (e.g. a status report cyclically initiated by the device)
  string activityId = 438006925;

  // Increased for each message sent by the device to enable reliable ordering of messages in the backend
  int32 deviceSequenceCounter = 96023102;

  // Message creation timestamp. Number of milliseconds that have elapsed since the Unix epoch (00:00:00 UTC on 1 January 1970)
  int64 timestamp = 55126294;

}
